applications/trilinos_application/python_scripts/trilinos_import_model_part_utility.py

```diff
@@ -29,17 +29,29 @@ def ExecutePartitioningAndReading(self):
         self.ImportModelPart()
 
     def ImportModelPart(self):
-        input_type = self.settings["model_import_settings"]["input_type"].GetString()
+        model_part_import_settings = self.settings["model_import_settings"]
```

Comment:
Yep please revert the changes in this file

Reply:
Yes, of course - just tried to get around the problem quickly. I hope this problem is now solved after I merged the master and accepted all incoming changes for this file.